File: app/src/main/java/com/mazanca/newrespiracao/model/RespiracaoEnergia.java
package com.mazanca.newrespiracao.model;

public class RespiracaoEnergia implements Respirar {

    @Override
    public String getNome() {
        return "Respiração Energizante";
    }

    @Override
    public long getTempoInspirar() {
        return 2;
    }

    @Override
    public long getTempoExpirar() {
        return 2;
    }

    @Override
    public long getNumeroDeCiclos() {
        return 5;
    }
}

File: app/src/main/java/com/mazanca/newrespiracao/model/RespiracaoEnergia.kt
package com.mazanca.newrespiracao.model;

import android.os.Parcelable
import kotlinx.parcelize.Parcelize

@Parcelize
data class RespiracaoEnergia( private val tempoPausaCustom: Long = 0): Respirar,Parcelable {
    override fun getNome()="Respiracao Energizante"
    override fun getTempoInspirar()=2L
    override fun getTempoExpirar()=2L
    override fun getNumeroDeCiclos()=8L
}


